### DISCUSSION TITLE
telemetry / tracking

### DISCUSSION THREAD

**Discussion by endquote** [WeblateOrg/weblate#9376] [General] [answered]
Does a self-hosted weblate instance report any data or telemetry back to the project, even if it's anonymous?

If so, is there a way to opt out?

**Comment by nijel** [comment, marked as answer]
No, unless you enable https://docs.weblate.org/en/latest/admin/support.html#discover-weblate

**Comment by endquote** [comment]
That's what I figured, thank you for confirming.
